### DISCUSSION TITLE
Adding notes

### DISCUSSION THREAD

**Discussion by pvinis** [jonaswinkler/paperless-ng#751] [Feature Requests]
In the Metadata section or in the Details section, it would be very useful to have a `Notes` textfield.
I very often need to have some notes there that I can see later. Things like "This is needed for X" or "Paid" or "Show this to X".

**Comment by jonaswinkler** [comment]
Well yes, there's #274, and it would allow notes text fields, among other things. However, that's a pretty big task and I've got limited time, sadly :(

I might wrap that one up by simply adding a notes field.

**Reply by pvinis** [reply]
Yea, that's why I made a separate one. Infinitely customizable metadata would be fun, but notes is an easy way to do that as a first version. Then everyone can just write what they want there.

**Reply by umoenks** [reply]
Absolutely! A simple notes field would already do a great job. This would make it possible to track some further devlopment of a document, as if you were physically taking notes on the document itself or a Post-It attached to it.

**Reply by luk6173** [reply]
Would like to see this feature also in Paperless-NG

**Reply by frank-from-cologne** [reply]
Yes, some more would like this, check [1226](https://github.com/jonaswinkler/paperless-ng/discussions/1226)

**Reply by philpagel** [reply]
Yes – that would be nice. Sometimes i have Notes or keywords that i would like to add for better "findability" that neither fit into the title nor warrant creating their own tag.
